### DISCUSSION TITLE
Document start / Document end

### DISCUSSION THREAD

**Discussion by fdzuJ** [toml-lang/toml#945] [Q&A] [answered]
Since it is outlined in README I guess it was discussed but I wasn't able to find any.
Why toml doesn't support a way to create several documents in single file?

**Comment by marzer** [comment, marked as answer]
- https://github.com/toml-lang/toml/issues/511
- https://github.com/toml-lang/toml/issues/583
- https://github.com/toml-lang/toml/issues/397
